wordlist.txt

```diff
@@ -4,6 +4,18 @@
 /phpinfo.php
 /info.php
 /phptest.php
+/etc/passwd
+/../etc/passwd
+/../../etc/passwd
+/../../../etc/passwd
+/../../../../etc/passwd
+/../../../../../etc/passwd
+/../../../../../../etc/passwd
+/../../../../../../../etc/passwd
+/../../../../../../../../etc/passwd
+/../../../../../../../../../etc/passwd
+/../../../../../../../../../../etc/passwd
```

Comment:
Aside from on reddit.com, I don't think you will stumble across many `/etc/passwd` files in the wild like this.

Reply:
I also haven't encountered. Maybe more luck with .htaccess?

Reply:
Yes, you are more likely to stumble across `.htaccess` files in practice.

Reply:
What do you grep for `.htaccess` ? There could be anything there as far as I know.

Reply:
Probably `RewriteEngine`. My goal with smith is not really to cover every single case, just the most common ones.